src/app/myworkflows/sidebar-accordion/github-apps-logs/github-apps-logs.component.ts

```diff
@@ -45,6 +40,20 @@ export class GithubAppsLogsComponent implements OnInit {
   showContent: 'table' | 'error' | 'empty' | null;
   isExpansionDetailRow = (i: number, row: Object) => row.hasOwnProperty('detailRow');
 
+  constructor(
+    @Inject(MAT_DIALOG_DATA) public data: string,
+    private lambdaEventsService: LambdaEventsService,
+    private matSnackBar: MatSnackBar
+  ) {
+    this.pipe = new DatePipe('en');
+    const defaultPredicate = this.dataSource.filterPredicate;
+    // tslint:disable-next-line:no-shadowed-variable
```

Comment:
it would be preferable to not have the shadowed variable. Actually, is still one there? I can't tell what it would be. In any case, there's a reason the linter is (was?) giving a warning, so, with exceptions, it's generally best not to suppress these warnings and instead deal with them.

Reply:
Fixed